basic_rules.md

```diff
@@ -257,10 +266,10 @@ Karta postępów ma być przede wszystkim pomocą dla gracza. Jednak warto pami
 
 ### 4.2. Zakupy i sprzedaż ekwipunku, korzystanie z płatnych usług<a name="uslugi"></a>
 
-Pomiędzy przygodami gracze mogą za posiadane złoto, bez poświęcania czasu wolnego:
-- kupować przedmioty z rozdziału Ekwipunek Podręcznika Gracza (po cenach tam podanych), komponenty do zaklęć (po cenach podanych przy zaklęciach) oraz [mikstury i inne przedmioty z tabel 3 i 7 w części II Zasad.](#tabela-mikstury)
+Pomiędzy przygodami gracze mogą:
+- kupować przedmioty z rozdziału Ekwipunek Podręcznika Gracza (po cenach tam podanych), komponenty do zaklęć (po cenach podanych przy zaklęciach) oraz [mikstury i inne przedmioty z tabel 3 i 4 w części II Zasad.](#tabela-mikstury)
```

Comment:
Why the change? Table 4 contains scrolls which require DT to be bought, and you removed mention of table 7 containing firearms.

Reply:
We have costs mentioned for each thing separately so we can get rid of mentioning it's free. 

As for table 7- industrial revolution was a mistake!

Reply:
@Mixxer5 totally agree! pistols are only of use if you intend to put them up someone's butt.
jokes aside, gonna revert the table number change